include/RE/B/BSShader.h

```diff
@@ -29,27 +29,51 @@ namespace RE
 		{
 		public:
 			// members
-			std::uint32_t      id;                  // 00
-			ID3D11PixelShader* shader;              // 08
-			ConstantGroup      constantBuffers[3];  // 10
-			std::int8_t        constantTable[64];   // 58
+			std::uint32_t                id;                  // 00
+			REX::W32::ID3D11PixelShader* shader = nullptr;    // 08
```

Comment:
`ID3D11PixelShader` and `ID3D11VertexShader` are forward declared in this file, this doesn't compile